pkg/machine/ignition/ignition.go

```diff
@@ -151,7 +151,7 @@ func (ign *DynamicIgnition) GenerateIgnitionConfig() error {
 		if ign.TimeZone == "local" {
 			tz, err = getLocalTimeZone()
 			if err != nil {
-				return err
+				return fmt.Errorf("error getting local timezone: %q", err)
```

Comment:
if we return an empty string the caller does not expect that and always creates an invalid localtime symlink now below now.

I would assume you need a tz != "" check around the ignition Link so we don't end up generateding broken link in the VM

Reply:
well other platforms use "" (windows for example); so i am not sure that this is bad or we set a a value like UTC as part of the default ?